scripts/dump-remote-schema.js

```diff
@@ -17,21 +17,29 @@ const httpConvectionLink = createHttpLink({
   uri: urljoin("https://convection-staging.artsy.net/api", "graphql"),
 })
 
-introspectSchema(httpConvectionLink).then(schema => {
-  fs.writeFileSync(
-    path.join(destination, "convection.graphql"),
-    printSchema(schema, { commentDescriptions: true })
+introspectSchema(httpConvectionLink)
+  .then(schema =>
+    fs.writeFileSync(
+      path.join(destination, "convection.graphql"),
+      printSchema(schema, { commentDescriptions: true })
+    )
   )
-})
+  .catch(e => {
+    throw e
+  })
```

Comment:
feels a bit weird to just pass the throw up from top level down and then back, but we're planning on removing this script eventually anyway